wrappers/verifier-registry.ts

```diff
@@ -0,0 +1,237 @@
+import {
```

Comment:
probably we should adapt the Contract interface for this wrapper (similar to what was applied to Sources Registry)?

Reply:
It does implement the Contract interface. Do you mean the queries function should be integrated into the contract class?

Reply:
More in the sense that anything using VerifierRegistry shouldn't be aware of Queries.
There should be instead functions for each operation in VerifierRegistry's API.